### DISCUSSION TITLE
why last rettweet 5000 is not added

### DISCUSSION THREAD

**Discussion by hizen1984** [IBAX-io/go-ibax#1816] [Q&A]
### Describe the bug

last rettweet task , 5000 points not added to scores???

### Reproduction

Reproducible

### System Info

```shell
browsers or application
```

### Logs

```shell
#IBAX #Airdrop End in 1 Day!
get on board before it ends! https://t.co/HWXFdt5ya6
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Mrt021** [comment]
Bu sorun herkeste mevcut anlık yoğunluktan olduğunu düşünüyorum

**Comment by Abdullahoda** [comment]
ilerleyen saatler de verilecektir

**Comment by Nesim581** [comment]
Aynen neden eklenmiyor

**Comment by emrecicek21** [comment]
Sanırım yoğunluktan dolayı puan gelmiyor

**Comment by kurdoglu24** [comment]
sabah 9 dan beri yapılmıyıor

**Comment by Ufuk145355** [comment]
last rettweet task , 5000 points not added to scores???

**Comment by ibaxio** [comment]
Please refer to this discussion:  #1813
